src/nxdoc/getting-started/best-practices/nuxeo-platform-feature-table.md

```diff
@@ -623,14 +623,14 @@ The tables below use the following notation:
     <td>&#10003;<br/>&#10003;</td>
     <td>&#10003;<br/>LTS&nbsp;2019<br/>&#10003;</td>
```

Comment:
Nuxeo Diff Picture should be N/A for Web UI
One new line should be added to this cells with "Arender connector" link and on description "document file comparison" (like comapring pdf, etc it now is a new thing and done with arender.